File: src/ProjectR/content/scripts/spells/TruePotential.lua
function GetTargetType()
  return Myself
end

function GetName()
  return "True Potential"
end

function GetDescription()
  return "Unlocks your full potential and increase all your stats."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .88
end

function GetDelay()
  return 0
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  for i = HP, SIL do
    a:BuffStat(i, .75)
  end
end

File: src/ProjectR.Scripting/content/scripts/spells/TruePotential.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class TruePotential : SpellScriptBase
    {
        public override string Name { get { return "True Potential"; } }
        public override string Description { get { return "Unlocks your full potential and increase all your stats."; } }

        public override TargetType TargetType { get { return TargetType.Myself; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 88; } }
        public override double Delay { get { return 0; } }

        public override void SpellEffect(ICharacter caster)
        {
            BuffAllStats(caster, .75);
        }
    }
}